misk/src/main/kotlin/misk/web/marshal/Json.kt

```diff
@@ -21,6 +22,7 @@ class JsonMarshaller<T>(val adapter: JsonAdapter<T>) : Marshaller<T> {
     }
   }
 
+  @Singleton
```

Comment:
should `@Singleton` on factory classes be an enforced / documented idiom?

Reply:
probably documented. We wanna say “everything that can be singleton should be singleton”